stubs/php-file-get-contents/README.md

```diff
@@ -0,0 +1,39 @@
+# How to run
+
+```console
+./run.php expired.badssl.com 443
+```
+
+# Dependencies
+
+ * php
```

Comment:
True

````
platform: OS X 10.10.5
runner: trytls 0.0.8 (CPython 2.7.10, OpenSSL 0.9.8zg)
stub: php 'stubs/php-file-get-contents/run.php'
ERROR expired certificate [reject expired.badssl.com:443]
      reason: stub exited with return code 1
      output: Could not open input file: stubs/php-file-get-contents/run.php
````

Default for the setting is true. So shall we ...?

 * Just document that if you have changed the default then you get the error
 *  Write a check in the code that returns `UNSUPPORTED` and prints the reeason if the default has been changed
 * Enforce the defaut in the code with `ini_set("allow_url_fopen", true);`

Any opinions?

Reference: https://secure.php.net/manual/en/filesystem.configuration.php